tests/php/Security/RememberLoginHashTest.php

```diff
@@ -110,20 +110,25 @@ public function testRenew($replaceToken)
 
         $member = $this->objFromFixture(Member::class, 'main');
 
-        RememberLoginHash::config()->set('replace_token_during_session_renewal', $replaceToken);
+        Deprecation::withNoReplacement(
+            fn() => RememberLoginHash::config()->set('replace_token_during_session_renewal', $replaceToken)
+        );
```

Comment:
That config is deprecated, so we need to wrap this to avoid deprecation warnings when we go hunting for stuff we need to update for the final minor release.